LICENSE.md

```diff
@@ -11,3 +15,5 @@ Redistribution and use in source and binary forms, with or without modification,
 * Neither the name of Spree Commerce Inc. nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
 
 _This software is provided by the copyright holders and contributors "as is" and any express or implied warranties, including, but not limited to, the implied warranties of merchantability and fitness for a particular purpose are disclaimed. In no event shall the copyright owner of contributors be liable for any direct, indirect, incidental, special, exemplary, or consequential damages (including, but not limited to, procurement of substitute goods or services; loss of use, data, or profits; or business interruption) however caused and on any theory of liability, whether in contract, strict liability, or tort (including negligence or otherwise) arising in any way out of the use of this software, even if advised of the possibility of such damage._
+
+Licenses for all other components used can be found in the file [Additional Licenses](https://github.com/solidusio/solidus/blob/Fix-License-Issues/additionallicenses.md).
```

Comment:
Can we use a relative link in markdown?

```suggestion
Licenses for all other components used can be found in the file [Additional Licenses](additionallicenses.md).
```

Reply:
Honestly I was not aware that git supports relative paths! 

Yes, makes sense and is absolutely indicated.